src/components/CollectionAlgorithmDecision.js

```diff
@@ -30,6 +30,11 @@ export default function CollectionAlgorithmDecision(props) {
         div({style: {fontSize: '1.5rem'}}, [
           span({id: `collection-${id}-date-label`, style: {paddingRight: '1%'}}, ['Date:']),
           span({id: `collection-${id}-date-value`, style: {fontWeight: 400}}, [!isNil(createDate) ? formatDate(createDate) : 'N/A'])
+        ]),
+        div({style: {fontSize: '1.5rem'}}, [
+          span({id: `collection-${id}-reason-label`, style: {paddingRight: '1%'}}, ['Reason:']),
```

Comment:
Comment with no action requested: I think padding with percentages are kind of risky and leave the code vulnerable to responsiveness problems in the future, personally. I see that this line of code is following the convention in this file, so I .. am not asking that you update it (or all of it), but just wanted to try pointing out in case others agree.

And if they do, i can help updating these paddings in a future pr?

Reply:
> And if they do, i can help updating these paddings in a future pr?

I agree Christine, please consider this task on the menu.

Reply:
Percentage and `em` are the de-facto measurements for responsive sizing though since the measurements are always relative to the parent container. Pixels are an absolute measurement and do not scale with screens of varying resolutions, so you're forced to designate media breakpoints in CSS.

Here's an article from the material-design team on the issues of using `px` outright, as well as how they used their own derived pixel measurement to avoid the issue

https://material.io/design/layout/pixel-density.html#pixel-density

Reply:
Created a new ticket to address this: [DUOS-2081](https://broadworkbench.atlassian.net/browse/DUOS-2081)